README.md

```diff
@@ -19,26 +19,18 @@ An overview of the advantages and differences as compared to the TF wrappers can
 This package is available on PyPi:
 
 ```bash
-# Normal tensorflow
+# Tensorflow
 pip install scikeras[tensorflow]
-
-# or tensorflow-cpu
-pip install scikeras[tensorflow-cpu]
 ```
 
-SciKeras packages TensorFlow as an optional dependency because there are
-several flavors of TensorFlow available (`tensorflow`, `tensorflow-cpu`, etc.).
-Depending on _one_ of them in particular disallows the usage of the other, which is why
-they need to be optional.
-
-`pip install scikeras[tensorflow]` is basically equivalent to `pip install scikeras tensorflow`
+Note that `pip install scikeras[tensorflow]` is basically equivalent to `pip install scikeras tensorflow`
 and is offered just for convenience. You can also install just SciKeras with
 `pip install scikeras`, but you will need a version of tensorflow installed at
 runtime or SciKeras will throw an error when you try to import it.
 
-The current version of SciKeras depends on `scikit-learn>=1.0.0` and `TensorFlow>=2.7.0`.
+The current version of SciKeras depends on `scikit-learn>=1.4.1post1` and `TensorFlow>=2.16.0`.
```

Comment:
Does it still depend on TF with keras 3, or is TF now optional?

Reply:
No, a lot of docs updates are necessary

Reply:
Fixed some references for now. Thank you for pointing them out.